src/test/java/org/apache/bcel/classfile/JDKClassDumpTestCase.java

```diff
@@ -71,17 +78,19 @@ private void testJar(final File file) throws Exception {
 
     @Test
     public void testPerformance() throws Exception {
-        final File javaLib = new File(System.getProperty("java.home") + "/lib");
-        javaLib.listFiles(file -> {
-            if (file.getName().endsWith(".jar")) {
-                try {
-                    testJar(file);
-                } catch (final Exception e) {
-                    fail(e.getMessage());
+        final File javaHome = new File(System.getProperty("java.home"));
+        Files.walkFileTree(javaHome.toPath(), new SimpleFileVisitor<Path>() {
```

Comment:
This duplicates existing test code, please refactor to use `org.apache.bcel.generic.JavaHome`.

Reply:
NM, just rebase on master, please. It's already done.